helm-branches.yaml

```diff
@@ -1,35 +1,51 @@
 deploy:quality:helm:branches:
   extends: .helm:deploy
   environment:
-    name: ${CI_COMMIT_REF_SLUG}
-    url: https://${DOMAIN}
+    name: review/$CI_BUILD_REF_NAME
+    url: https://${CI_COMMIT_REF_SLUG}.${DOMAIN_QUALITY}
     on_stop: stop:quality
   variables:
     CI_COMMIT_TAG: ${CI_COMMIT_SHORT_SHA}
     GOOGLE_KEY: ${GOOGLE_KEY_QUALITY}
     CLUSTER_NAME: ${CLUSTER_NAME_QUALITY}
     CLUSTER_ZONE: ${CLUSTER_ZONE_QUALITY}
-    SECRET_YAML: ${SECRET_YAML_QUALITY}
+    #SECRET_YAML: ${SECRET_YAML_FEATURE}
     NAMESPACE: ${NAMESPACE_QUALITY}
     ENVIRONMENT: feature
     APP_NAME: ${CI_COMMIT_REF_SLUG}
   before_script:
     - |
+      # use a specific CI/CD variable if SECRET_BRANCH is specified in gitlab-ci
+      if [ -n "${SECRET_BRANCH}" ]; then
+        export SECRET_YAML=${!SECRET_BRANCH}
+      # otherwise, use default SECRET_YAML_FEATURE CI/CD variable
+      else
+        export SECRET_YAML=${SECRET_YAML_FEATURE}
+      fi
+    - echo ${SECRET_YAML}
+
+    - |
+      # use a specific domain if DOMAIN_BRANCH is specified in gitlab-ci
       if [ -n "${DOMAIN_BRANCH}" ]; then
         db=${CI_COMMIT_REF_SLUG}.${DOMAIN_BRANCH}
+      # otherwise, use default DOMAIN_QUALITY
       else
         db=${CI_COMMIT_REF_SLUG}.${DOMAIN_QUALITY}
       fi
       export DOMAIN=$( eval echo \$db )
-    - echo ${DOMAIN}
+    - echo https://${DOMAIN}
+
     - |
+      # use a specific host if APP_NAME_BRANCH is specified in gitlab-ci
```

Comment:
don't forget to also write it in the README

Reply:
It's there, line 635 :)